ci_framework/roles/devscripts/tasks/cleanup.yml

```diff
@@ -54,7 +54,11 @@
 - name: Cleanup the devscripts working directory.
   become: true
   ansible.builtin.file:
-    path: "{{ cifmw_devscripts_config.working_dir }}"
+    path: >-
+      {{
+        cifmw_devscripts_config.working_dir |
+        default('/home/dev-scripts')
```

Comment:
I thought it was set in the defaults already?

Reply:
Yes, it is configured as `defaults`... thanks looks like I was cautious